spacy/lang/punctuation.py

```diff
@@ -39,8 +39,20 @@
         r"(?<=[{al}{q}])\.(?=[{au}{q}])".format(
             al=ALPHA_LOWER, au=ALPHA_UPPER, q=CONCAT_QUOTES
         ),
-        r"(?<=[{a}]),(?=[{a}])".format(a=ALPHA),
+        r"(?<=[{a}])(?:{p})(?=[{a}])".format(a=ALPHA, p=PUNCT),
         r"(?<=[{a}])(?:{h})(?=[{a}])".format(a=ALPHA, h=HYPHENS),
         r"(?<=[{a}0-9])[:<>=/](?=[{a}])".format(a=ALPHA),
     ]
 )
+
```

Comment:
I decided to put this here rather than copying it into the three language-specific directories because it is potentially relevant to any language with variable stress written in Cyrillic, of which there are many (e.g. several Finno-Ugric languages). If any of these languages are supported by spaCy in the future, it will be useful to have this definition here.